test/backend/CMakeLists.txt

```diff
@@ -10,6 +10,26 @@ file(GENERATE
   INPUT ${CMAKE_CURRENT_SOURCE_DIR}/test.py
   )
 
```

Comment:
Is `conftest.py` used anywhere at this point?

Reply:
Actually I do not know how `conftest.py` is used. @gongsu832 Can you explain a bit about this?

Reply:
It's used by pytest. In this particular case, it's used to add a pytest option to remove the test progress report that prints a lot of hash, dot, etc. that clutter up the outputs.

Reply:
@sstamenova Based on gong's explaination, I think we should not remove conftest.py here. You still think we need changes here?

Reply:
No, thanks for looking into it.